src/palindrom.js

```diff
@@ -487,12 +487,20 @@ var Palindrom = (function() {
     reconnectionFn(function bootstrap(json) {
       palindrom.reconnector.stopReconnecting();
 
-      palindrom.queue.reset(palindrom.obj, json);
-
       if (palindrom.debug) {
         palindrom.remoteObj = JSON.parse(JSON.stringify(json));
       }
 
+      if(!palindrom.isObjectProxified) { // first time the state is being set
+        palindrom.observe(json);
+      }
+      palindrom.unobserve();
+
+      if(!palindrom.isObjectProxified) { // first time the state is being set, just set the versioning correctly
+        palindrom.queue.reset(palindrom.obj, palindrom.obj);
```

Comment:
Could you elaborate? Is the Queue not ready to receive a patch just after `new` ?